saw-remote-api/CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
   LLVM verification, this field is optional. For MIR verification, this field
   is required if the `"elements"` value is empty and optional if the
   `"elements"` value is non-empty.
+* The old `"tuple"` `setup value` has been renamed to `"struct"`. This better
+  reflects its intended purpose of representing struct values. There is now a
+  new `"tuple"` `setup value` that is only used to represent MIR tuples.
```

Comment:
Just to confirm - this is not a user-facing change, right?  It looks like the Python API was using the name `struct`/`StructVal` for the thing that was internally called "tuple", and only the internal name has changed.

I was going to ask whether users who forgot to update their scripts would get a useful error message, but if the Python API is actually unchanged, then it's a moot point.

Reply:
Correct, the Python API remains unchanged. The only way this could break anything is if there are any other libraries out there somewhere that interface with the SAW remote API, but I'm not aware of any such libraries.